tests/store/api/conftest.py

```diff
@@ -252,7 +271,7 @@ def fixture_re_sequenced_sample_store(
     store_sample = helpers.add_sample(
         internal_id=sample_id,
         is_tumour=False,
-        application_type="tgs",
+        application_type=str(PrepCategory.READY_MADE_LIBRARY),
```

Comment:
```suggestion
        application_type=PrepCategory.READY_MADE_LIBRARY.value,
```

Reply:
Cool, was wondering what the correct way to do this was :)